high_school/models.py

```diff
@@ -19,3 +19,15 @@ class HighSchool(models.Model):
 
     def __str__(self):
         return self.school_name
+
+
+class Program(models.Model):
+    high_school = models.ForeignKey(HighSchool, on_delete=models.CASCADE)
+    name = models.CharField(max_length=100, blank=False, null=False)
+    code = models.CharField(unique=True, max_length=20, blank=False, null=False)
```

Comment:
will the program code be same for a specific program for different high schools?

Reply:
"Each program has a unique code, except those in Specialized High Schools". About 8 schools have "NO CODE" in the program code and we don't need those. Except that all of the ones that have code are unique.